drawBot/aliases.py

```diff
@@ -0,0 +1,12 @@
+import os
+from typing import Sequence
+
+Point = Size = tuple[float, float]
+Points = Sequence[Point]
+SomePath = os.PathLike | str
+CMYKColor = float | tuple[float, ...]
+# used for output values which are always 4 elements
+RGBAColorTuple = tuple[float, float, float, float]
+RGBColor = float | tuple[float, ...]
+BoundingBox = tuple[float, float, float, float]
+Transform = tuple[float, float, float, float, float, float]
```

Comment:
Should we consider calling this `TransformTuple`, do make it easier to distinguish from fontTools' Transform (which is a named tuple, and can be used here, too)?

Reply:
fixed with c8c13714728abc1f7ace2662a104c50181d9d08e